PostPairsTrading.md

```diff
@@ -29,14 +22,25 @@ The concept we're referring to is **cointegration** (although there are other me
 > 💡 Which should not be confused with correlation; cointegration is a statistical property of two-time series, indicating a long-term relationship between them despite short-term fluctuations. Cointegrated series move together over time, sharing a common stochastic drift. On the other hand, correlation measures the strength and direction of the linear relationship between two variables at a specific point in time. While correlation captures the degree of association between variables, cointegration reflects a deeper, underlying relationship that persists over time.
 
 Hence, we're interested in **cointegrated assets**, which are assets that exhibit the following characteristics:
+* They have a similar trend, meaning the difference between both assets maintains a constant mean, and this difference fluctuates around that same mean.
+* This inherent relationship persists in the long run, meaning that our series is not dependent on time.
+Before delving into this search for cointegrated pairs, let me introduce you the tick architecture.
+
+## What is the Tick Architecture?
 
-1. They have a similar trend, meaning the difference between both assets maintains a constant mean, and this difference fluctuates around that same mean.
+The tick architecture is designed to handle high-frequency trading data efficiently. At its heart is the tickerplant (TP), a crucial component responsible for receiving and timestamping incoming data, then broadcasting it to other components such as the real-time database (RDB) and historical database (HDB). The TP ensures that data is distributed in a timely manner, allowing for real-time analytics and decision-making. The RDB stores recent data for quick access, while the HDB archives older data for long-term storage and analysis. Additionally, the feedhandler plays a vital role by interfacing with external data sources, making sure that the TP receives accurate and up-to-date information. This architecture guarantees seamless data flow and rapid access to both real-time and historical data, making it ideal for high-frequency trading applications.

```

Comment:
Maybe we should explain (in a single line perhaps) what it _is_ at a higher level and how it fits our "project" before going deep into its components

Reply:
I think I just copied your intro to the tick architecture, but I find it ok if you want to extend it. We don't talk much about the performance and the general benefits, perhaps it would be nice to briefly discuss it right here.